src/main/webapp/index.jsp

```diff
@@ -13,63 +13,63 @@
+    </style>
+    <link rel="stylesheet" href="/css/main.css">
+  </head>
+
+  <body>
+    <nav>
+      <a id="navTitle" href="/">CodeU Chat App</a>
+      <% if(request.getSession().getAttribute("user") != null){ %>
+        <a>Hello
+          <%= request.getSession().getAttribute("user") %>!</a>
+        <a href="/conversations">Conversations</a>
+        <a href="/profiles">Profiles</a>
+        <a href="/activityfeed">Activity</a>
+        <%if (request.getSession().getAttribute("user").equals("EmilyArroyo")
       || request.getSession().getAttribute("user").equals("AlexandriaStorm")
       || request.getSession().getAttribute("user").equals("AnaVitoriadoValleCosta")
       || request.getSession().getAttribute("user").equals("KevinWorkman")
       || request.getSession().getAttribute("user").equals("GavinLifrieri")) { %>
-       <a href="/admin">Administration</a>
-        <% } else{ %>
-          <a href="/login">Login</a>
-        <% } %>
-           <% } else{ %>
-      <a href="/login">Login</a>
-    <% } %>
-    <a href="/about.jsp">About</a>
-  </nav>
-
-  <div id="container">
-    <div
-      style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px;">
+          <a href="/admin">Administration</a>
+          <% } else{ %>
+            <a href="/login">Logout</a>
+            <% } %>
+              <% } else{ %>
+                <a href="/login">Login</a>
+                <% } %>
+                  <a href="/about.jsp">About</a>
+    </nav>
 
-      <h1>CodeU Chat App</h1>
-      <h2>Welcome!</h2>
+    <div id="container">
+      <div style="width:100%; margin-left:auto; margin-right:auto;
+        margin-top: 50px; text-align: center; color: #222429;">
+        <h1>Team 18's Google CodeU Project</h1>
+        <img src="/images/google.gif" style="width: 400px;"/>
+        <h2>Welcome to our Translation Chat Web App!</h2>
+        <body> We created this open source project as a part of the Google CodeU Summer Program.
 
-      <ul>
-        <li><a href="/login">Login</a> to get started.</li>
-            <% if(request.getSession().getAttribute("user") != null){ %>
-               <li>View the <a href="/profiles">profiles</a> page to see user profiles.<li>
-               <li>Go to the <a href="/conversations">conversations</a> page to
-                   create or join a conversation.</li>
-            <%  } %>
-        <li>View the <a href="/about.jsp">about</a> page to learn more about the
-            project.</li>
-      </ul>
+        <h2>Features:</h2>
+        <ul>
+          Translation Feature - We used Google Translate's APIs to create a chat application
+            that automatically translate's all messages from friends to the user's preferred language.
+          This eliminates the language barrier for users to communicate easily with different people.
+          Each user declares their main language upon registration and then the website shows all of their
+          conversations in that language.
```

Comment:
Does the translation part work yet?

Reply:
No, I guess we're still waiting on Gavin to make a PR, it looks like he made commits with it in it though.